### DISCUSSION TITLE
Can't create and train model

### DISCUSSION THREAD

**Discussion by eranmahalu** [d8ahazard/sd_dreambooth_extension#953] [General]
Hi friends, how are you doing? I have an error in Dreambooth, thought you might have stumbled upon it
"if checkpoint_file is None or not os.path.exists(checkpoint_file[0]):
TypeError: 'CheckpointInfo' object is not subscriptable"
this happens when I try to create a model. please help

**Comment by Arilziem** [comment]
Hello. That's not really enough information to help you. Could be a VRAM issue. Could you post the startup info (versions, startup parameters; see [bug report template](https://github.com/d8ahazard/sd_dreambooth_extension/blob/main/.github/ISSUE_TEMPLATE/bug_report.md)) and more than just two lines of that error?
